peer/brontide.go

```diff
@@ -730,6 +730,21 @@ func (p *Brontide) Start() error {
 		}
 	}
 
+	data, err := p.cfg.PeerDataStore.RetrievePeerData(p.cfg.Addr.
+		IdentityKey)
+
+	if err != nil {
+		return fmt.Errorf("unable to retrieve peer backup data: "+
+			"%v", err)
+	}
+
+	if err := p.writeMessage(&lnwire.YourPeerStorage{
+		Blob: data,
+	}); err != nil {
+		return fmt.Errorf("unable to send "+
```

Comment:
So no delay here? Even though it was included in the spec @ProofOfKeags?

Reply:
The spec simply says that we may delay things to rate limit. IMO rate limiting is a decision we should make but the delay is not necessarily the angle we have to go after. I am suggesting we defer that decision to a later commit. Here we just need to get the bones in place.

Reply:
Also yeah I wouldn't put the delay here at all. The idea of the delay would be to withhold the ack of a peer storage message not delay the retrieval.